### PR TITLE
Created travisTest script to enable testing daily build

### PR DESCRIPTION
**Pull request by evelinec**
- Created travisTest script to include the defined automation test steps for this guide; this is needed so we can run daily build test for this guide.
 - Reference info for the `set -euxo pipefail` flag : https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
- addressed issue #36

### CODE REVIEW

**Comment by evelinec** [comment]
Pending on the fix for #38 to pass the test.

**Comment by gkwan-ibm** [comment]
@evelinec please take a look on https://github.com/OpenLiberty/guide-getting-started/pull/78/files, see how the getting guide fix the travisTest.sh. Andrew and me spent sometime to find out.
- search "version" instead of "on java" to support different JDK
- format of the travisTest.sh
- not everything can be moved into the travisTest.sh because TRAVIS_TEST_RESULT does not available inside travisTest.sh

**Comment by evelinec** [comment]
@gkwan-ibm I examined the changes made in the getting-started guide PRs for the automation tests. First, the fix for getting the jdk11 version info is good! Thank you. It actually fixed this issue we had https://github.com/openliberty/guides-common/issues/217 (if it's working, I'll further verify). 

However, there are couple problems:
- the test steps put in the .travis.yml file are not going to be run by the daily build test automation
- the getting-started guide have the release, build and jdk collection information running twice in the daily build runs

Basically, the daily build test script needs the jdk11 version info fix and it does the actions to send the runtime server logs to the artifactory. 

@andrewdes is informed and working on creating some PRs for the getting started guide, with the above mentioned points.

As with this PR. I also fixed the similar issues accordingly here. The tests are running fine. I'll test the daily build automation with these changes and report the status here.

**Comment by evelinec** [comment]
@gkwan-ibm can you review this PR? Thanks

**Comment by evelinec** [comment]
Formatted the `status=...` command in the last commit. The `docker...` command is one line with all the parameters, they can't be separated. @gkwan-ibm pls review again. Thanks.

**Comment by gkwan-ibm** [comment]
can the docker command to be as following?
```
docker run -d --name rest-app \
  -p 9080:9080 -p 9443:9443 \
  -v /home/travis/build/OpenLiberty/guide-docker/finish/target/liberty/wlp/usr/servers:/servers \
  -u `id -u` ol-runtime
```

**Comment by evelinec** [comment]
Sure. Still have one parameter that's wouldn't fit in one line. It's updated in the last commit. Thanks.

**Comment by evelinec** [comment]
@gkwan-ibm pls review. Thanks.

**Comment by evelinec** [comment]
Note, there's no change in the readme adoc in this PR, thus no need to verify the UI.
